cookbook/core/extend_flyte/task_template.py

```diff
@@ -0,0 +1,81 @@
+"""
+.. _task_template:
+
+Custom Container Task Plugins
+-----------------------------
+
+Task Templates are useful as they: 
```

Comment:
Let't just say these are the advantages in comparison to a custom behavior plugin.